dev-python/dpath/dpath-2.1.6.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{4..11} )
```

Comment:
please drop unsupported python versions (anything older then 3.10). Could you also test with 3.12? Adding 3.12 now will save work later.